managed/services/telemetry/telemetry.go

```diff
@@ -106,18 +106,35 @@ func (s *Service) Run(ctx context.Context) {
 	defer ticker.Stop()
 
 	doSend := func() {
+		var settings *models.Settings
+		err := s.db.InTransaction(func(tx *reform.TX) error {
+			var e error
+			if settings, e = models.GetSettings(tx); e != nil {
+				return e
+			}
+			return nil
+		})
+		if settings.Telemetry.Disabled {
+			s.l.Info("disabled via settings")
```

Comment:
what record will be dropped into log stream? Please show an example. Is it clear enough to determine that this is from telemetry sub-system?

Reply:
yes, there is component
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/91831719/180392802-538d11f1-66ba-4dbe-8089-904063abad0f.png">